src/common/snippets/src/lowered/pass/cleanup_loop_offsets.cpp

```diff
@@ -36,7 +37,7 @@ bool CleanupLoopOffsets::run(lowered::LinearIR& /*linear_ir*/,
             // Note: Finalization offsets before the Result can be safely disregarded
             // TODO: Need verify that Buffers on the inputs doesn't have other consumers (other Loops)
             //       and this Loop doesn't have Buffer on other outputs.
-            if (is_type<ov::op::v0::Result>(next_node)) {
+            if (is_type<op::Result>(next_node) || is_type<ov::op::v0::Result>(next_node)) {
```

Comment:
Shall we get rid of v0::Result usage then?

Reply:
done!